modules/core/localizer.js

```diff
@@ -391,5 +391,35 @@ export function coreLocalizer() {
         return code;  // if not found, use the code
     };
 
+    localizer.floatFormatter = (locale) => {
+        if (!('Intl' in window && 'NumberFormat' in Intl &&
+              'formatToParts' in Intl.NumberFormat.prototype)) {
+            return (number) => number.toString();
+        } else {
+            return (number) => number.toLocaleString(locale);
+        }
+    };
+    localizer.floatParser = (locale) => {
+        // https://stackoverflow.com/a/55366435/4585461
+        const polyfill = (string) => parseFloat(string, 10);
```

Comment:
1. There is no second parameter for [parseFloat](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/parseFloat).
2. Would it make sense to trim the string before parsing here, so the behaviour here is consistent with the full-fledged implementation below (line 416)?

```suggestion
        const polyfill = (string) => parseFloat(string.trim());
```